zarr/storage.py

```diff
@@ -65,14 +65,7 @@
     from zarr.codecs import Zlib
     default_compressor = Zlib()
 
-# Find which function to use for atomic replace
-if sys.version_info >= (3, 3):
-    from os import replace
-elif sys.platform == "win32":  # pragma: no cover
-    from osreplace import replace
-else:  # pragma: no cover
-    # POSIX rename() is always atomic
-    from os import rename as replace
+from os import replace
```

Comment:
Can we combine this with the `from os import ...` above?

Reply:
done.